frc_characterization/newproject/__init__.py

```diff
@@ -114,18 +115,38 @@ def deployProject():
         else:
             cmd = "deploy"
 
+        no_jdk_warning = lambda: (
+            messagebox.showwarning(
+                "Warning!",
+                "You do not appear to have a " + str(datetime.now().year) + " wpilib JDK installed. " +
+                "If your installed JDK is the wrong version then the deploy will fail."
+            )
+        )
+
         if os.name == "nt":
+            process_args = [
+                os.path.join(
+                    STATE.project_path.get(),
+                    "characterization-project",
+                    "gradlew.bat",
+                ),
+                cmd,
+                "--console=plain",
+            ]
+
+            # This path is correct *as of* wpilib 2020
+            # Prior to 2020 the path was C:/Users/Public/frcYEAR/jdk
+            jdk_path = os.path.join(
+                "C:", os.sep, "Users", "Public", "wpilib", str(datetime.now().year), "jdk"
```

Comment:
This is wrong for Windows installs not on C:\.

Reply:
Fixed.